libmy/spooldir.c

```diff
@@ -99,13 +99,13 @@ spooldir_init(const char *path)
 
 	s->dname_active = ubuf_init(UBUFSZ);
 	ubuf_add_cstr(s->dname_active, dname);
-	ubuf_add_cstr(s->dname_active, "/active");
```

Comment:
The spooldir and zonefile code is unused in wdns, and should be left alone in this change set.